code/game/machinery/adv_med.dm

```diff
@@ -411,7 +410,7 @@
 		extra_font = (occupant.getBrainLoss() < 1 ?"<font color='blue'>" : "<font color='red'>")
 		dat += "[extra_font]\tApprox. Brain Damage %: [occupant.getBrainLoss()]<br>"
 
-		dat += "Paralysis Summary: [round(occupant.AmountParalyzed() / 10)] seconds left.<br>"
+		dat += "Drunkenness Severity: [round(occupant.get_drunkenness() / 10)]<br>"
```

Comment:
Why the / 10 here?

Reply:
drunkness is in deciseconds, so it shows it in seconds.

Reply:
which is true, but it was just an accident from my part, it should show the real value

Reply:
fixed in https://github.com/ParadiseSS13/Paradise/pull/18926/commits/bd5eb9688096fe752ba4d200fa7204ae9ffb21c8